src/components/tasks/card/index.tsx

```diff
@@ -41,30 +41,54 @@ const Card: FC<Props> = ({
     const timeLeft = localEndsOn.valueOf() - currentDate.valueOf();
     return !statusNotOverDueList.includes(cardDetails.status) && timeLeft <= 0;
   }
-
+  
   function stripHtml(html: string) {
     const tmp = document.createElement('DIV');
     tmp.innerHTML = html;
     return tmp.textContent || tmp.innerText || '';
   }
-
+  
```

Comment:
Could we look into whitespace issue here please

Reply:
I don't if I understand fully, but one line is deliberately left after each function. Tell me if I am understanding it wrong